modules/oe_paragraphs_media/config/install/core.entity_view_display.paragraph.oe_text_feature_media.default.yml

```diff
@@ -2,19 +2,32 @@ langcode: en
 status: true
 dependencies:
   config:
+    - field.field.paragraph.oe_text_feature_media.field_oe_link
     - field.field.paragraph.oe_text_feature_media.field_oe_media
     - field.field.paragraph.oe_text_feature_media.field_oe_plain_text_long
     - field.field.paragraph.oe_text_feature_media.field_oe_text_long
     - field.field.paragraph.oe_text_feature_media.field_oe_title
     - paragraphs.paragraphs_type.oe_text_feature_media
   module:
-    - image
```

Comment:
Why removed?

Reply:
Because it isn't used. Drupal removed it after fresh site installation.